docs/peer_review_response.md

```diff
@@ -81,6 +81,34 @@ _LGTM_ <br>yay :)
+3. *You could also just have the name of the treat be the primary key instead of the sku because it looks like most of the skus and names are the same if not similar as it would reduce the amount of data held and simplify the table a bit*
+    - Non-issue. SKUs are easier to reference over names because they have no spaces. Plus, the treats table itself doesn't grow often, so having an extra column for names is not much of an issue for data storage.
+4. *Have the name just be the primary key to creature_types as it seems to uniquely identify each column and it is what is being referenced to from foreign keys from other tables.*
+    - Suggestion taken. The primary key for creature_types is now name.
+5. *In the set up for the creatures table it seems you already have default values set up for happiness, health, and hunger, but are not using them. I do not know if you want these default values to be the starting values but, if you do, then do not specify those columns in the insert into command.*
+    - Default values are now taken advantage of when breeding creatures.
+6. *For many of the id columns you have it set up so that they are generated by default so do not specify them in the insert into command as they will generate for you and it will simplify the sql statement*
+    - Ids are manually specified in `schema.sql` for testing purposes.
+7. *The name column in the creatures table seems to uniquely identify each row and it is the column referenced to by most foreign keys from other tables, so just make that the primary key for the table.*
+    - [pending]
+8. *I think it would be sufficient if your ids in your tables were regular, smaller integers rather than bigints*
+    - This doesn't really make any difference performance-wise. We will never hit the upper 32-bit or 64-bit limit regardless.
+9.  *In the API spec, I would have the sku in the CartItem class as well as allowing a list of CArtItems to be passed in just to simplify the function of adding items to a cart and so that it may only be called once.*
+    - No longer relevant. Shop rework merges the set_item_quantity and checkout endpoints, resulting in one call per purchase of an item.
+10. *Instead of having success booleans, such as play_success and feed_success, passed to the user, you could just throw errors to warn the user that doing this action would be pointless.*
+    - Fixed by implementing appropriate HTTP status codes.
+11. *The sku and the name of each treat seem to be the same if not similar so it may not be necessary to display both to the user when they call the GET catalog function*
+    - [pending]
+12. *Delete user should be a POST I believe instead of specifying it as a DELETE as our classifications are mainly PUT, POST, or GET.*
+    - Point taken. "delete" is already specified in the endpoint URL anyways, so I suppose specifying the HTTP method as DELETE isn't necessary. Changed to POST.
```

Comment:
Using 'DELETE' follows REST practices. Considering we haven't been penalized yet for using 'DELETE', I would stick with it. We can remove '/delete' from the url if that's preferred. 
Also, where does it say our main classifications are PU, POST, and GET? I don't remember seeing that